test/unit/services/setupSecrets.test.ts

```diff
@@ -0,0 +1,72 @@
+const { expect } = require('chai');
+const { cloneDeep } = require('lodash');
+const config = require('config');
+const proxyquire = require('proxyquire');
+
+const modulePath = 'app/server/services/setupSecrets';
+
+let mockConfig: any = {};
+
+describe(modulePath, () => {
+  describe('#setup', () => {
+    beforeEach(() => {
+      mockConfig = cloneDeep(config);
+    });
+
+    it('should set config values when secrets path is set', () => {
+      mockConfig.secrets = {
+        sscs: {
+          'sscs-cor-redis-connection-string': 'sessionValue',
+          'sscs-cor-redis-access-key': 'redisValue',
+          'sscs-cor-idam-client-secret': 'idamValue',
+          'sscs-s2s-secret': 'osPlacesValue'
+        }
+      };
+
+      // Update config with secret setup
+      const setupSecrets = proxyquire(modulePath,
+        { config: mockConfig });
+      setupSecrets.setupKeyVaultSecrets();
```

Comment:
I think this will fix it
```suggestion
      setupSecrets();
```